detectron2/modeling/box_regression.py

```diff
@@ -102,12 +102,15 @@ def apply_deltas(self, deltas, boxes):
         pred_w = torch.exp(dw) * widths[:, None]
         pred_h = torch.exp(dh) * heights[:, None]
 
-        pred_boxes = torch.zeros_like(deltas)
-        pred_boxes[:, 0::4] = pred_ctr_x - 0.5 * pred_w  # x1
-        pred_boxes[:, 1::4] = pred_ctr_y - 0.5 * pred_h  # y1
-        pred_boxes[:, 2::4] = pred_ctr_x + 0.5 * pred_w  # x2
-        pred_boxes[:, 3::4] = pred_ctr_y + 0.5 * pred_h  # y2
-        return pred_boxes
+        return torch.cat(
+            [
+                (pred_ctr_x - 0.5 * pred_w).unsqueeze(dim=-1),  # x1
+                (pred_ctr_y - 0.5 * pred_h).unsqueeze(dim=-1),  # y1
+                (pred_ctr_x + 0.5 * pred_w).unsqueeze(dim=-1),  # x2
+                (pred_ctr_y + 0.5 * pred_h).unsqueeze(dim=-1),  # y2
+            ],
+            dim=-1,
+        ).flatten(1)
```

Comment:
fyi, I'm only waiting to first get the fix https://github.com/pytorch/pytorch/pull/45828 into pytorch so we don't have to do this hack